sdk/src/main/java/io/radar/sdk/model/RadarUser.kt

```diff
@@ -206,29 +217,9 @@ class RadarUser(
             val trip = RadarTrip.fromJson(obj.optJSONObject(FIELD_TRIP))
 
             return RadarUser(
-                id,
-                userId,
-                deviceId,
-                description,
-                metadata,
-                location,
-                geofences,
-                place,
-                insights,
-                beacons,
-                stopped,
-                foreground,
-                country,
-                state,
-                dma,
-                postalCode,
-                nearbyPlaceChains,
-                segments,
-                topChains,
-                source,
-                fraud.proxy,
-                trip,
-                fraud.mocked
+                id, userId, deviceId, description, metadata, location, geofences, place, insights, beacons,
+                stopped, foreground, country, state, dma, postalCode, nearbyPlaceChains, segments, topChains, source,
+                fraud.proxy, trip, fraud.mocked
```

Comment:
I actually like it the old way here.

Reply:
@jrtibbetts it's nice, but it causes a line-limit exception. The real fix would be to change how many parameters the constructor can take, but for now this is the best option.

Reply:
yeah detekt is a static analysis tool for Kotlin